### ISSUE TITLE
[Bug]: cannot export image in shared canvas due to CORS error

### ISSUE THREAD

**Issue by mangkoran** [tldraw/tldraw#3986]
Just now I also encountered this issue when I'm trying to export shared canvas.
```
Access to fetch at 'https://assets.tldraw.xyz/uploads/REDACTED' from origin 'https://www.tldraw.com' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'null' that is not equal to the supplied origin. Have the server send the header with a valid value, or, if an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```
Edit: Nvm I think this only affect if I'm trying to export images. I can export other object just fine. So I guess I will open a new issue.

_Originally posted by @mangkoran in https://github.com/tldraw/tldraw/issues/3140#issuecomment-2177747267_

**Comment by mangkoran** [comment]
Nvm I found the issue is due to I use Decentraleyes. Now I have turned it off for tldraw.com and it works.

![image](https://github.com/tldraw/tldraw/assets/57625126/2a697c2c-5924-4708-8158-6fb3db2df3dc)

**Comment by mangkoran** [comment]
Nvm turns out I encountered this again. So I have 2 images on my page. I can copy and export one of them but not the other.

**Comment by mimecuvalo** [comment]
Thanks for reporting! We have recently changed how our CORS fetching works so it seems like this started hitting a particular configuration with Cloudflare and caching.

I modified the Cloudflare Browser Cache TTL to not override and set it to 4 hours and instead to "Respect Existing Headers" since we have a worker specifically to handle this.

This is fixed now.

**Comment by mimecuvalo** [comment]
(You might have to clear your local browser cache potentially)

**Comment by mangkoran** [comment]
I just tried and it seems it works now. Thank you very much!
